### PR TITLE
Add settings to configure text for arrows

### LINKED ISSUE #3: Setting to set text for the back/forward button
I think it would be helpful if there was a setting in order to set the text that is displayed for the button. For instance, something like:
`vscode-back-forward-button.backwards-text = '<'`
`vscode-back-forward-button.forwards-text = '>'`

This allows customizing the buttons to the user's liking. For instance, you could do:
 * Triangles (◀ ▶)
 * Tails on the arrows (<- ->)
 * Angle brackets (〈 〉)
 * Left/Right arrows (← →)

### PR DESCRIPTION
**Pull request by datho7561**
Added `backAndForwardButtons.backText`
and `backAndForwardButtons.forwardText`,
which allow the user to set what text is displayed
for the buttons in the settings menu.

BackAndArrowButtonSettings

Upgraded the dependencies because I was
having trouble installing them through `npm install`.

Closes #3

Signed-off-by: David Thompson <davidethompson@me.com>

### CODE REVIEW

**Comment by datho7561** [comment]
~~I am going to attempt to make this feature work without upgrading all the dependencies, but until then I am going to leave this as a draft PR.~~ I couldn't get `npm install` to work without first running `npm update`.

File: package.json [hunk 1]
```diff
@@ -19,7 +19,22 @@
   ],
   "main": "./out/extension",
   "contributes": {
-    "commands": []
+    "commands": [],
```
Comment (datho7561) [inline package.json:22]: wait a second, I should remove this line, since there are no commands contributed

File: package.json [hunk 2]
```diff
@@ -29,10 +44,10 @@
     "test": "npm run compile && node ./node_modules/vscode/bin/test"
   },
   "devDependencies": {
+    "@types/mocha": "^2.2.42",
```
Comment (datho7561) [inline package.json:47]: I removed the tests, so I should also remove mocha.

(or actually add tests, probably the better route)

**Comment by grimmerk** [comment]
@datho7561, thanks for this PR. I'm sorry that I did not notice this PR and will review it soon and submit it accordingly if it is OK.

**Comment by datho7561** [comment]
Okay. Thank you!

**Pull request merged**